src/components/permission/PermissionSpec.tsx

```diff
@@ -0,0 +1,224 @@
+import { CommandData, PermissionData } from "@site/src/components/permission/Permissions";
+
+export const vanillaCommands: CommandData = {
+  permissionPrefix: "minecraft.command",
+  commands: [
+    { name: "advancement", aliases: [], opLevel: 2 },
+    { name: "attribute", aliases: [], opLevel: 2 },
+    { name: "ban", aliases: [], opLevel: 3 },
+    { name: "ban-ip", aliases: [], opLevel: 3 },
+    { name: "banlist", aliases: [], opLevel: 3 },
+    { name: "bossbar", aliases: [], opLevel: 2 },
+    { name: "clear", aliases: [], opLevel: 2 },
+    { name: "clone", aliases: [], opLevel: 2 },
+    { name: "damage", aliases: [], opLevel: 2 },
+    { name: "data", aliases: [], opLevel: 2 },
+    { name: "datapack", aliases: [], opLevel: 2 },
+    { name: "debug", aliases: [], opLevel: 3 },
+    { name: "defaultgamemode", aliases: [], opLevel: 2 },
+    { name: "deop", aliases: [], opLevel: 3 },
+    { name: "difficulty", aliases: [], opLevel: 2 },
+    { name: "effect", aliases: [], opLevel: 2 },
+    { name: "enchant", aliases: [], opLevel: 2 },
+    { name: "execute", aliases: [], opLevel: 2 },
+    { name: "fill", aliases: [], opLevel: 2 },
+    { name: "fillbiome", aliases: [], opLevel: 2 },
+    { name: "forceload", aliases: [], opLevel: 2 },
+    { name: "function", aliases: [], opLevel: 2 },
+    { name: "gamemode", aliases: [], opLevel: 2 },
+    { name: "gamerule", aliases: [], opLevel: 2 },
+    { name: "give", aliases: [], opLevel: 2 },
+    { name: "help", aliases: [], opLevel: 0 },
+    { name: "item", aliases: [], opLevel: 2 },
+    { name: "jfr", aliases: [], opLevel: 4 },
+    { name: "kick", aliases: [], opLevel: 3 },
+    { name: "kill", aliases: [], opLevel: 2 },
+    { name: "list", aliases: [], opLevel: 0 },
+    { name: "locate", aliases: [], opLevel: 2 },
+    { name: "loot", aliases: [], opLevel: 2 },
+    { name: "me", aliases: [], opLevel: 0 },
+    { name: "msg", aliases: ["w", "tell"], opLevel: 0 },
+    { name: "op", aliases: [], opLevel: 3 },
+    { name: "pardon", aliases: [], opLevel: 3 },
+    { name: "pardon-ip", aliases: [], opLevel: 3 },
+    { name: "particle", aliases: [], opLevel: 2 },
+    { name: "perf", aliases: [], opLevel: 4 },
+    { name: "place", aliases: [], opLevel: 2 },
+    { name: "playsound", aliases: [], opLevel: 2 },
+    { name: "publish", aliases: [], opLevel: 4 },
+    {
+      name: "random",
+      aliases: [],
+      opLevel: "Yes without sequence argument, No with sequence argument.",
```

Comment:
Now that we have proper fields we don't need to reuse them for multiple purposes